packages/manager/apps/pci-private-registry/src/pages/api-url/APIUrl.page.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_COLOR_INTENT,
+  ODS_TEXT_LEVEL,
+} from '@ovhcloud/ods-components';
+import {
+  OsdsButton,
+  OsdsClipboard,
+  OsdsModal,
+  OsdsSpinner,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useGetAllRegistries } from '@/api/hooks/useRegistry';
+
+export default function ApiUrlPage() {
+  const { t } = useTranslation();
+
+  const navigate = useNavigate();
+  const onClose = () => navigate('..');
+
+  const { projectId, registryId } = useParams();
+  const { data: registries, isPending } = useGetAllRegistries(projectId);
+  const registry = registries?.find((r) => r.id === registryId);
+
+  return (
+    <OsdsModal
+      onOdsModalClose={onClose}
+      headline={t('private_registry_harbor_api_title')}
+    >
+      <div className="mt-5">
+        {isPending ? (
+          <OsdsSpinner
+            inline
+            size={ODS_SPINNER_SIZE.md}
+            className="block text-center mt-5"
+          />
+        ) : (
+          <>
+            <OsdsText
+              color={ODS_TEXT_COLOR_INTENT.text}
+              level={ODS_TEXT_LEVEL.body}
+              size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+            >
+              {t('private_registry_harbor_api_url')}
+            </OsdsText>
+            <OsdsClipboard
+              className="mt-4"
+              aria-label="clipboard"
+              value={registry?.url}
+            >
+              <span slot="success-message">{t('common_clipboard_copied')}</span>
+            </OsdsClipboard>
```

Comment:
You can use the manager-component Clipboard, you won't need to add another translation key.